### DISCUSSION TITLE
JSX Style Guide 공유합니다.

### DISCUSSION THREAD

**Discussion by dev-heeseok** [dev-heeseok/web-react#15] [Announcements]
@kimkmin357 @DeukHan 
[Airbnb  JSX Style Guild](https://airbnb.io/javascript/react/) 를 공유합니다.
비슷한 스타일로 개발을 진행하면 의미 전달과 클린 코드를 개발하는데 도움이 될것 같네요.

추가로 위의 문서는 Github 를 통해서도 공유되고 있습니다. 
[Airbnb Github](https://github.com/airbnb/javascript/tree/master/react)

**Comment by DeukHan** [comment]
좋은 정보 공유 감사합니다.
